dbt/include/duckdb/macros/materializations/external.sql

```diff
@@ -7,11 +7,14 @@
   {%- set read_location = adapter.external_read_location(location, rendered_options) -%}
 
   -- set language - python or sql
+  -- i have to learn general about python models
   {%- set language = model['language'] -%}
 
   {%- set target_relation = this.incorporate(type='view') %}
 
   -- Continue as normal materialization
+  -- Why do we have temp and intermediate relation? 
+  -- What does load_cached_relation do?
```

Comment:
Built-in dbt method; dbt will do the equivalent of caching the DB catalog metadata it loaded during the run so as to minimize the number of round-trips it does to the database (less relevant for DuckDB, more relevant for e.g. MotherDuck.)

Reply:
I will have to take a look into MotherDuck here because I still don't understand how this fits here and what the places are where I have to be careful not to overlook the functionality. So please, if you see something that is against MotherDuck logic, make me aware of that.